server/jetstream_sourcing_scaling_test.go

```diff
@@ -345,45 +433,83 @@ func TestStreamSourcingScalingSourcingManyBenchmark(t *testing.T) {
 		DiscardNewPerSubject: false,
 	})
 	require_NoError(t, err)
-
+	c.waitOnStreamLeader(globalAccountName, "sourcing")
 	sl := c.streamLeader(globalAccountName, "sourcing")
-	mset, err := sl.GlobalAccount().lookupStream("sourcing")
-	require_NoError(t, err)
+	fmt.Printf("Sourcing stream created leader is *** %s ***\n", sl.Name())
 
-	start := time.Now()
+	expectedSeq := make([]int, numSourced)
+
+	//var steppedDown bool
```

Comment:
leftover  comment

Reply:
Cleaned up. Those were part of a previous change to the test that would artificially trigger a leader change towards the start of the test, which actually would hide some of the possible failure scenarios.